entrypoint.sh

```diff
@@ -68,13 +68,12 @@ fi
 
 if [ -z "$ORYX_DISABLE_TELEMETRY" ] || [ "$ORYX_DISABLE_TELEMETRY" == "false" ]; then
     url="https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}/jobs"
-    json=$(curl -X GET "${url}")
-    startTime=${json#*Build*/appservice-build@*,}
-    startTime=$(echo "${startTime}"| sed 's/,/\n/g' | grep "started_at" | awk '{print $2}' | sed -n '1p')
-    endTime=${json#*Build*/appservice-build@*,}
-    endTime=$(echo "${endTime}" | sed 's/,/\n/g' | grep "completed_at" | awk '{print $2}' | sed -n '1p')
-    export GITHUB_ACTIONS_BUILD_IMAGE_PULL_START_TIME=$startTime
-    export GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME=$endTime
+    jobs=$(curl -X GET "${url}")
+    appserviceBuildJob=${jobs#*Build*/appservice-build@*,}
+    startedAtTime=$(echo "${appserviceBuildJob}" | sed 's/,/\n/g' | grep "started_at" | awk '{print $2}' | sed -n '1p' | tr -d '"')
+    completedAtTime=$(echo "${appserviceBuildJob}" | sed 's/,/\n/g' | grep "completed_at" | awk '{print $2}' | sed -n '1p' | tr -d '"')
+    export GITHUB_ACTIONS_BUILD_IMAGE_PULL_START_TIME=$startedAtTime
+    export GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME=$completedAtTime
 fi
```

Comment:
Before fix:
```
 echo $GITHUB_ACTIONS_BUILD_IMAGE_PULL_START_TIME
"2020-06-24T21:06:27.000Z"
echo $GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME
"2020-06-24T21:06:59.000Z"
```

After fix:
```
echo $GITHUB_ACTIONS_BUILD_IMAGE_PULL_START_TIME
2020-06-24T21:06:27.000Z
echo $GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME
2020-06-24T21:06:59.000Z
```

We just removed quotes since they were causing issues when parsing in Oryx build script generator code

cc @qianz2